tests/queries/0_stateless/02180_group_by_lowcardinality.sql

```diff
@@ -0,0 +1,14 @@
+create database if not exists d_group_by_lowcardinality;
```

Comment:
>emmmm， I visit some fail task. message is 'table is not exists', how can I fix it?

The problem is that the test is running in parallel and some of runs may remove table in parallel.

You can fix this by simply removing creating database (and using qualified db.table names in the tests), since each test already have it's own random database.

Reply:
ok. I try to do it.

Reply:
You forgot to remove qualified table names, i.e .`s/d_group_by_lowcardinality.t_group_by_lowcardinality/t_group_by_lowcardinality`

Reply:
.... ok.